src/main/java/org/fundacionjala/coding/yury/movies/Customer.java

```diff
@@ -0,0 +1,57 @@
+package org.fundacionjala.coding.yury.movies;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This class Customer.
+ */
+public class Customer {
+    private String name;
+    private List<Rental> listRentals = new ArrayList<>();
+
+    /**
+     * @param string this name movie
+     */
+    public Customer(String string) {
+        this.name = string;
+    }
+
+    /**
+     * @param rental values of movie
+     */
+    public void addRental(Rental rental) {
+        listRentals.add(rental);
+    }
+
+    /**
+     * @return message
+     */
+    public String statement() {
```

Comment:
This method is doing 3 things, it is calculating the totalAmount, totalFrequentRenterPoints and it is printing.
Please split it.